main/FatesHistoryInterfaceMod.F90

```diff
@@ -2508,7 +2508,7 @@ subroutine update_history_dyn(this,nc,nsites,sites,bc_in)
                hio_lai_si                           => this%hvars(ih_lai_si)%r81d )
 
    ! If we don't have dynamics turned on, we just abort these diagnostics
-   if (hlm_use_ed_st3.eq.itrue) return
```

Comment:
I'm working on breaking up the history calls so that we are not processing variables that are not desired or active. My take, is that by removing this line, the user can now view phenologically relevant output?

Reply:
To be honest, I removed this because I wanted to see the computed forest structure properties in the output, and forgot to uncomment it back for the PR. But you are right, this return may disable variables that are now going to change due to phenology. Perhaps we could move the phenology-related variables before the if and keep the others after the if?

Reply:
my intention in the PR that cleans up history was to enable diagnostics that show forest structure and states, and to turn off flux variables (with st3).  So, if you don't have a strong preference for this PR, lets just keep the return in there to make testing more straight forward?